### ISSUE TITLE
Filter/Search doesn't appear to work

### ISSUE THREAD

**Issue by ghost** [Gruntfuggly/vscode-journal-view#14]
When I search for a text string that I know is in a note that note is not showing up in any search result view.

**Comment by Gruntfuggly** [comment]
Yep - looks like it's not searching notes - I'm away for a few days, but I'll sort it out next week.

**Comment by Gruntfuggly** [comment]
Hmm - I just tried this and it seems to work fine on my mac at home. When I tried it on my work machine it didn't seem to work. I'll try it again on Monday.

**Comment by timharris777** [comment]
Just discovered the journal and journal-view plugins. The search prompts me for text and then does nothing. I'm assuming it's not working the way it's supposed to

**Comment by Gruntfuggly** [comment]
Hi - unfortunately, due to some personal issues, I haven't had time to pay any attention to this extension for a while.  When I get some time, I'll definitely see if I can work out why it's not working.
